pass-fedora-client/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.dataconservancy.pass</groupId>
+    <artifactId>pass-client</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+  <artifactId>pass-fedora-client</artifactId>
+  <name>pass-fedora-client</name>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+  <dependencies>
+	<dependency>
+      <groupId>org.dataconservancy.pass</groupId>
+      <artifactId>pass-model</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    
+	<dependency>
+      <groupId>org.dataconservancy.pass</groupId>
+      <artifactId>pass-json-adapter</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    
+    <dependency>
+      <groupId>org.dataconservancy.pass</groupId>
+      <artifactId>pass-client-util</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    
+	<dependency>
+	  <groupId>org.fcrepo.client</groupId>
+	  <artifactId>fcrepo-java-client</artifactId>
+	  <version>${fcrepo-java-client.version}</version>
+	</dependency>
+    
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <version>${logback.version}</version>
+      <optional>true</optional>
+    </dependency>
+	
+    <dependency>
```

Comment:
If JUnit is specified as a `<dependency>` in the parent pom, it doesn't need to be doubly-included here.

Reply:
I removed the junit dependency from the module it the build failed - what did I do wrong? I've changed the parent pom.